src/mdx-pages/blog/videojs-8-and-vhs-3.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: Video.js 8 and Video.js HTTP Streaming 3
+author:
+  name: Pat O'Neill
+  github: misteroneill
+date: 2022-08-22 14:49:43
+tags:
+---
+
+<div class="note">
+  <p><b>NOTE:</b> For details on migrating from Video.js 7.x to 8.x, please see <a href="/guides/videojs-7-to-8/">our migration guide</a>!</p>
+</div>
+
+We are very excited to announce that Video.js 8.0 and Video.js HTTP Streaming (VHS) 3.0 are available!
+
+While these releases are constituted of a lot of housecleaning tasks, there are some changes of interest to users of the Video.js platform.
+
+## Contributors
+
+Before we dive into the software changes, open source doesn't work without _people_. Huge thanks to everyone who contributed pull requests to these releases!
+
+* [@alex-barstow](https://github.com/alex-barstow)
+* [@gesinger](https://github.com/gesinger)
+* [@gjanblaszczyk](https://github.com/gjanblaszczyk)
+* [@gkatsev](https://github.com/gkatsev)
+* [@harisha-swaminathan](https://github.com/harisha-swaminathan)
+* [@hugorogz](https://github.com/hugorogz)
+* [@kchang-brightcove](https://github.com/kchang-brightcove)
+* [@mister-ben](https://github.com/mister-ben)
+* [@misteroneill](https://github.com/misteroneill)
+* [@phloxic](https://github.com/phloxic)
+* [@roman-bc-dev](https://github.com/roman-bc-dev)
+
+## What's Changed?
+### Video.js
+
+The [complete CHANGELOG is available](https://github.com/videojs/video.js/blob/TODO/CHANGELOG.md), but here are some highlights:
+
+* No longer transpiled into ES5 for compatibility with older browsers like Internet Explorer
+* Updated to VHS 3.0
+* Enabled the `sourceset` event by default, `enableSourceset: false` can be used to turn this off.
+* Added new `TitleBar` component, which is not visible in the UI by default _(see below for details)_. As a result, we moved the `BigPlayButton` component to the center of the player by default
+* `addClass` and `removeClass` methods can now be given multiple class names
+* Clicking the playback rate button now opens the menu rather than changing the playback rate
+* Deprecated many old top-level utility methods in favor of utility methods stored on objects _(see below for details)_
+* Invalid event types will now throw errors instead of log warnings
+* Change `addRemoteTextTrack`'s `manualCleanup` option to default to `false`
+* Removed the `videojs.extend()` function
+* Removed the `firstplay` event
+* Removed the `retryOnError` option and made this behavior the default
+* Removed the ability to set `aria-*`, `role`, and `type` attributes via the `props` argument of `createEl` methods
+* Removed remaining references and logic related to Flash and SWF
+* Remove fallbacks for missing flexbox support
+* Removed IE-specific code
+
+Overall, the minfied and gzipped size of Video.js is **reduced by around 3%**. More savings are likely available through further removal of dead code.
```

Comment:
Perhaps word this more as a commitment to continue to optimise rather than just an acknowledgement that it is possible.